zip-0000.rst

```diff
@@ -163,51 +166,91 @@ SHOULD be added to the ZIP metadata indicating the original authorship
 ZIP Editors
 -----------
 
-The current ZIP Editors are Daira Emma Hopwood, representing the Electric Coin
-Company, and Deirdre Connolly, representing the Zcash Foundation. Both
-can be reached at zips@z.cash . The current design of the ZIP Process
-dictates that there are always at least two ZIP Editors: one from the
-Electric Coin Company and one from the Zcash Foundation. Additional Editors may
-be selected by consensus among the current Editors.
+The current ZIP Editors are:
+- Daira Emma Hopwood and Jack Grigg (representing the Electric Coin
+Company)
+- teor and Conrado Gouvea (representing the Zcash Foundation).
 
+All can be reached at zips@z.cash. The current design of the ZIP Process
+dictates that there are always at least two ZIP Editors: at least one from the
+Electric Coin Company and at least one from the Zcash Foundation.
 
-ZIP Editor Responsibilities & Workflow
---------------------------------------
+Additional Editors may be selected by consensus among the current Editors.
+An Editor may be removed or replaced by consensus among the current Editors.
+However, if the other ZIP editors have consensus, an Editor can not prevent their own removal.
+Any Editor can resign by informing the other Editors in writing.
+
+Whenever the ZIP editors change, the new ZIP editors should:
+- review this ZIP to make sure it reflects current practice,
+- update the owners of this ZIP,
+- review ZIP editor access to the `ZIPs repository <https://github.com/zcash/zips>`,
+- update the <zips@z.cash> email alias, and
+- invite new editors to the Zcash Community Forums, and the #zips channel on Discord.
+
+If it has been at least 12 months since the last ZIP editor change, the ZIP editors should:
```

Comment:
```suggestion
However, if the other ZIP Editors have consensus, an Editor can not prevent their own removal.
Any Editor can resign by informing the other Editors in writing.

Whenever the ZIP Editors change, the new ZIP Editors should:
- review this ZIP to make sure it reflects current practice,
- update the owners of this ZIP,
- review access to the `ZIPs repository <https://github.com/zcash/zips>`,
- update the <zips@z.cash> email alias, and
- invite new editors to the Zcash Community Forums, and the #zips channel on Discord.

If it has been at least 12 months since the last ZIP Editor change, the ZIP Editors should:
```